XYZDrivers/src/main/webapp/index.html

```diff
@@ -1,10 +1,44 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>Start Page</title>
+        <title>Home Page</title>
         <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <link rel="stylesheet" type="text/css" href="node_modules/bootstrap/dist/css/bootstrap.min.css" />
     </head>
     <body>
-        <h1>Hello World!</h1>
+        <nav class="navbar navbar-default">
+            <div class="container-fluid">
+              <div class="navbar-header">
+                <a class="navbar-brand" href="#">XYZ Drivers Association</a>
+              </div>
+            </div><!-- /.container-fluid -->
+        </nav>
+        <div class="row text-center">
+            <h1>Welcome to the driver's Association</h1>
+            <h3>Please select an action</h3>
+            
+            <form method="GET" action="home">
+            
+                <div class="form-group">
+                    <label for="userType">Type of User:</label>
+                    <select name="userType">
+                        <option value="admin">Admin</option>
+                        <option value="user">User</option>
+                    </select>
+                </div>
+
+                <div class="form-group">
+                    <label for="actionType">Type of Action</label>
+                    <select name="actionType">
+                        <option value="login">Login</option>
+                        <option value="register">Register</option>
```

Comment:
Not sure we want someone to able to register as an admin. There's only one supposed to be one admin account I think.